recipes/pushover/webview.js

```diff
@@ -6,10 +6,8 @@ const _path = _interopRequireDefault(require('path'));
 
 module.exports = Ferdium => {
   const getMessages = () => {
-    const totalMessages = document.querySelector(
-      '#app_0 > div.app_counter',
-    ).innerHTML;
-    Ferdium.setBadge(totalMessages);
+    const totalMessages = document.getElementsByClassName("message_row").length;
+    Ferdium.setBadge(totalMessages)
```

Comment:
This line is missing a semicolon which will probably break the prettier formatting check:

```suggestion
    Ferdium.setBadge(totalMessages);
```